tutorials/elixir-phoenix-on-google-app-engine.md

```diff
@@ -48,37 +56,238 @@ If you have not yet installed Elixir and Phoenix, do so:
 
-3.  Run the app with the following command:
+        # Configure your database
+        config :hello, Hello.Repo,
+          username: "my_name",
+          password: "XXXXXXXX",
+          database: "hello_dev",
+          hostname: "localhost",
+          pool_size: 10
+
+4.  Create the development database with the following command:
+
+        mix ecto.create
+
+5.  Run the app with the following command:
 
         mix phx.server
 
     This compiles your server and runs it on port 4000.
 
-4.  Visit [http://localhost:4000](http://localhost:4000) to see the Phoenix
+6.  Visit [http://localhost:4000](http://localhost:4000) to see the Phoenix
     welcome screen running locally on your workstation.
 
+### Create and test a development database
+
+Next you will populate a simple development database and verify that your
+Phoenix app can access it.
+
+1.  Create a simple schema.
+
+        mix phx.gen.schema User users name:string email:string
+
+2.  Migrate your development database.
+
+        mix ecto.migrate
+
+3.  We'll add some very simple code to show that the application can access
+    the database, by querying for and displaying the number of user records.
+    Open `lib/hello_web/controllers/page_controller.ex` and rewrite
+    the `index` function as follows:
+
+        def index(conn, _params) do
+          value = Hello.Repo.aggregate(Hello.User, :count, :id)
+          conn
+          |> assign(:value, value)
+          |> render("index.html")
+        end
+
+4.  Display the value by inserting the following into the
+    `lib/hello_web/templates/page/index.html.eex` template:
+
+        <p>Number of users: <%= @value %></p>
+
+5.  Recompile and run the app
+
+        mix phx.server
+
+6.  Visit [http://localhost:4000](http://localhost:4000) to verify that your
+    new code is running. You can log into your database and add new rows, and
+    reload the page to verify that the count has changed.
+
+For more information on using Ecto to access a SQL database, see the
+[Phoenix Ecto guide](https://hexdocs.pm/phoenix/ecto.html).
+
+## Create a production database in Cloud SQL
+
+In this section, you will create your "production" database using Cloud SQL, a
+fully-managed database service providing PostgreSQL and MySQL in the cloud. If
+you already have a database hosted elsewhere, you may skip this section, but
+you may need to ensure your production configuration is set up to connect to
+your database.
+
+### Create a Cloud SQL instance
+
+First you will create a new database in the cloud.
+
+1.  Create a Cloud SQL instance named `hellodb` with a Postgres database
+    by running the following command:
+
+        gcloud sql instances create hellodb --region=us-central \
+          --database-version=POSTGRES_9_6 --tier=db-g1-small
+
+    You may choose a region other than `us-central` if there is one closer to
+    your location.
+
+2.  Get the _connection name_ for your Cloud SQL instance by running the
```

Comment:
Consider reducing the number of italicized terms. For example, I don't think the italics are needed here for connection name.